src/tasks/mixin/liaison_mixin.py

```
@@ -189,7 +189,7 @@ def special_chat_detect():
             chat_box = self.find_feature("chat_icon_dark") or self.find_feature("chat_icon_2")
 
             if chat_box:
-                self.send_key_up("w")
+                self.send_key_up("w")  # 确认使用send_key：释放方向键（W为移动键，不属于可配置热键）
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)